libs/user-identity-service/src/test/java/uk/gov/di/ipv/core/library/service/UserIdentityServiceTest.java

```diff
@@ -1352,7 +1375,7 @@ void shouldReturnEmptyWhenMissingDrivingPermitProperty() throws Exception {
                         USER_ID_1, "test-sub", Vot.P2, emptyContraIndicators);
 
         // Assert
-        assertNull(credentials.getDrivingPermitClaim());
+        assertTrue(credentials.getDrivingPermitClaim().isNull());
```

Comment:
Could this have stayed as `assertNull`?

Reply:
`assertNull` is not the same as `isNull` called on a JsonNode (which is what the credentials. getDrivingPermitClaim() now returns`